androidbrowserhelper/publish.gradle

```diff
@@ -0,0 +1,90 @@
+apply plugin: 'maven-publish'
+apply plugin: 'com.jfrog.bintray'
+
+version '0.1.0-alpha4'
+group 'me.bandarra'
+
+
+Properties properties = new Properties()
+properties.load(project.rootProject.file('local.properties')
+        .newDataInputStream())
+
+// build.gradle
+task sourcesJar(type: Jar) {
+    from android.sourceSets.main.java.source
+    classifier = 'sources'
+}
+task javadoc(type: Javadoc) {
+    source = android.sourceSets.main.java.source
+    ext.androidJar = "${android.sdkDirectory}/platforms/${android.compileSdkVersion}/android.jar"
+    classpath += (project.files(android.getBootClasspath().join(File.pathSeparator))
+            + files(ext.androidJar))
+    options.links("https://developer.android.com/reference/");
+    failOnError false
+}
+afterEvaluate {
+    javadoc.classpath += files(android.libraryVariants.collect
+            { variant ->
+                variant.javaCompile.classpath.files
+            })
+}
+task javadocJar(type: Jar, dependsOn: javadoc) {
+    classifier = 'javadoc'
+    from javadoc.destinationDir
+}
+artifacts {
+    archives javadocJar
+    archives sourcesJar
+}
+
+bintray {
+    user = properties.getProperty("bintray.user")
+    key = properties.getProperty("bintray.apikey")
+    pkg {
+        repo = 'android-browser-helper'
+        name = 'androidbrowserhelper'
+        userOrg = 'andreban'
+        desc = 'A Companion Library for Web On Android'
+        licenses = ['Apache-2.0']
+        vcsUrl = 'https://github.com/andreban/android-browser-helper'
+        githubRepo = 'andreban/android-browser-helper'
+        githubReleaseNotesFile = 'README.md'
```

Comment:
So the release notes for all releases will be checked into this repo somewhere? (That seems like a good thing, just wondering).